pkg/errmetrics/map.go

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package errmetrics
+
+import (
+	"fmt"
+	"syscall"
+
+	"github.com/cilium/ebpf"
+	"golang.org/x/sys/unix"
+)
+
+type Map struct {
+	*ebpf.Map
+}
+
```

Comment:
fwiw, I've been using the same approach in other places.
 * https://github.com/cilium/tetragon/blob/a1205b06bd5b2fb2868013e24bbfcde91c12f062/pkg/policyfilter/map.go#L25-L28
 * https://github.com/cilium/tetragon/blob/a1205b06bd5b2fb2868013e24bbfcde91c12f062/pkg/cgtracker/cgtracker.go#L33-L36

It's basically an easy way to provide some helper methods so that the map can be used by the CLI (`tetra`)